src/library/ButtonGroup/__tests__/ButtonGroup.spec.js

```diff
@@ -0,0 +1,104 @@
+/* @flow */
+import React from 'react';
+import { shallow } from 'enzyme';
+import { mountInThemeProvider } from '../../../../utils/enzymeUtils';
+import Button from '../../../library/Button';
+import ButtonGroup from '../ButtonGroup';
+import examples from '../../../website/app/demos/ButtonGroup/examples';
+import testDemoExamples from '../../../../utils/testDemoExamples';
+
+const defaultProps = {
+  'aria-label': 'Test',
+  children: <Button>Content</Button>,
+  onChange: jest.fn(),
+  onClick: jest.fn()
+};
+
+function shallowButtonGroup(props = {}) {
+  const buttonGroupProps = {
+    ...defaultProps,
+    ...props
+  };
+  return shallow(<ButtonGroup {...buttonGroupProps} />);
+}
+
+function mountButtonGroup(props = {}) {
+  const buttonGroupProps = {
+    ...defaultProps,
+    ...props
+  };
+
+  return mountInThemeProvider(<ButtonGroup {...buttonGroupProps} />);
+}
+
+describe('ButtonGroup', () => {
+  testDemoExamples(examples);
+
+  it('renders', () => {
+    const buttonGroup = shallowButtonGroup();
+
+    expect(buttonGroup.exists()).toEqual(true);
+  });
+
+  describe('event handlers', () => {
+    let button, buttonGroup;
+
+    beforeEach(() => {
+      defaultProps.onClick.mockReset();
+      defaultProps.onChange.mockReset();
+    });
+
+    describe('when no mode', () => {
+      beforeEach(() => {
+        [, buttonGroup] = mountButtonGroup();
+        button = buttonGroup.find('button');
+      });
+
+      it('calls onClick handler', () => {
+        button.simulate('click');
+
+        expect(defaultProps.onClick).toHaveBeenCalled();
```

Comment:
i think it would also be good to assert that onChange was not called in this case